backend/README.md

```diff
@@ -0,0 +1,19 @@
+# How to run
```

Comment:
We already have #14. Maybe look at that PR and add anything you did additionally as a change request?